app/pages/home.cjsx

```diff
@@ -79,7 +79,7 @@ module.exports = React.createClass
     document.documentElement.classList.remove 'on-home-page'
 
   lastFourProjects: ->
-    @props.user.get("project_preferences", page_size: 4, sort: '+updated_at')
+    @props.user.get("project_preferences", page_size: 4, sort: 'updated_at')
```

Comment:
Yeah - I was surprised the index wasn't being used and I investigated the query, it didn't have an order by at all from the sort param scope being set.